backend/composer/services/cs_ingestion/cs_ingestion_services.py

```diff
@@ -0,0 +1,234 @@
+        circuit_type = map_circuit_type_uri_with_model(statement[CIRCUIT_TYPE][0]) if has_prop(statement[CIRCUIT_TYPE]) else CircuitType.UNKNOWN
+        sex =  get_value_or_none(Sex, statement[SEX])
+        functional_circuit_role = get_value_or_none(FunctionalCircuitRole, statement[FUNCTIONAL_CIRCUIT_ROLE])
+        #some values from neurondm's phenotype field are not mapped in composer db. Add only the first one founded in composer db, if any
+        phenotypes_list = get_valid_phenotypes(statement)
+        phenotype = pick_first_phenotype(phenotypes_list, statement)
+        # other phenotypes includes 3 predicates, we will only store hasProjectionPhenotype which belongs to the last element of the other_phenotypes list from neurondm
+        try:
+            projection_phenotype = ProjectionPhenotype.objects.get(ontology_uri=statement[OTHER_PHENOTYPE][-1])
+        except:
+            projection_phenotype = None
+
+        # create the statement
+      
+        connectivity_statement, created = ConnectivityStatement.objects.get_or_create(
+            reference_uri__exact=reference_uri,
+            defaults={
+            "sentence": sentence, "knowledge_statement": knowledge_statement,"reference_uri": reference_uri, "origin": origin, "destination": destination, "destination_type": destination_type, "circuit_type": circuit_type, "sex": sex, "functional_circuit_role": functional_circuit_role, "phenotype": phenotype, "projection_phenotype": projection_phenotype
+            }
+        )
+        # add the many to many fields: path, species, provenances, notes
+        if created:
+            species = Specie.objects.filter(ontology_uri__in=statement[SPECIES])
+            connectivity_statement.species.add(*species)
+            #TODO add display_order criteria on neurondm update
+            vias = (Via(connectivity_statement = connectivity_statement, anatomical_entity=AnatomicalEntity.objects.filter(ontology_uri=via[ENTITY_URI])[0], type=via[TYPE]) for via in statement[VIAS])
+            Via.objects.bulk_create(vias)
+            provenances_list = statement[PROVENANCE][0].split(", ") if has_prop(statement[PROVENANCE]) else []
+            provenances = (Provenance(connectivity_statement = connectivity_statement, uri=provenance) for provenance in provenances_list)
+            Provenance.objects.bulk_create(provenances)
+            # only 5 statements have a note_alert but they were all filtered out since at least one of their anatomical entities was not found in composer db
+            if has_prop(statement[NOTE_ALERT]):   
+                Note.objects.create(connectivity_statement = connectivity_statement, user=User.objects.get(username="system"), type=NoteType.ALERT, note=statement[NOTE_ALERT][0])
+            connectivity_statement.save()
+
+            # transition to compose_now
+            do_transition_to_compose_now(sentence)
+
```

Comment:
@Anto8819 do we also need to transition the statement to a new state?

Reply:
@zsinnema you mean to the EXPORTED state, right? yep, I forgot that one, I'll add it

Reply:
@Anto8819 yep

Reply:
done